### PR TITLE
New args params

### PR DESCRIPTION
**Pull request by olehkrupko**

### CODE REVIEW

**Comment by olehkrupko** [comment]
There is a black error that can be avoided using two simple options on the screenshots below.

Error message:
swagger2locustio/__main__.py:17:0: R0915: Too many statements (52/50) (too-many-statements)

Option 1: less variables — less statements
Снимок экрана 2020-05-26 в 18 05 16

Option 2: similar to option 1, bu file is opened on higher level to avoid code repeats
Снимок экрана 2020-05-26 в 18 02 35

Any recommendations are very welcome as options above are not the only ones, but are the simplest solution and also reduce amount of variables used (maximum amount is 15)

File: swagger2locustio/utils.py [hunk 1]
```diff
@@ -7,33 +7,35 @@
...
 
 
-def log_diff(start, end, results_path):
+def log_diff(start, end, results_path, overwrite_confirm, overwrite_deny):
     """Function: log difference"""
 
-    changed_files_user_check(start["files"], end["files"], results_path)
+    if not overwrite_confirm:
+        changed_files_user_check(start["files"], end["files"], results_path, overwrite_deny)
+        end = log_result(results_path)
```
Comment (olehkrupko) [inline swagger2locustio/utils.py:31]: log_result(results_path) function is called as log_diff() input ( it happens in the end of __main__.py's main() function), so file structure is being parsed twice in a short period.

Is simple and easy to understand solution such as current one alright?

File: swagger2locustio/__main__.py [hunk 2]
```diff
@@ -74,6 +91,12 @@ def main():
     if tags and not_tags:
         raise ValueError("Both `tags` and not `not_tags` arguments specified")
 
+    if args.overwrite_confirm and args.overwrite_deny:
+        raise ValueError("Both `--overwrite-confirm` and not `--overwrite-deny` arguments specified")
```
Comment (viktor-svirsky) [inline swagger2locustio/__main__.py]: I think "not" is redundant here

File: swagger2locustio/utils.py (hunk 1 above)
Comment (viktor-svirsky) [reply swagger2locustio/utils.py:31]: - file structure is being parsed twice in a short period. Does it make sense to keep all parsed files data in some internal dict, and do not parse file more than once?

File: swagger2locustio/__main__.py (hunk 2 above)
Comment (olehkrupko) [reply swagger2locustio/__main__.py]: Yes, thank you

File: swagger2locustio/utils.py (hunk 1 above)
Comment (olehkrupko) [reply swagger2locustio/utils.py:31]: We already do it and discussed the situation in Skype. Such code fits our needs: marking as solved

**Review by viktor-svirsky — APPROVED** [review]

**Pull request merged**